service/resource/deployment/deployment.go

```diff
@@ -0,0 +1,44 @@
+package deployment
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/azuretpr"
+
+	"github.com/giantswarm/azure-operator/service/key"
+)
+
+const (
+	clusterIDParameterName     = "clusterID"
+	clusterSetupDeploymentName = "cluster-setup"
+	clusterSetupTemplate       = "cluster_setup.json"
+	templateBaseURI            = "https://raw.githubusercontent.com/giantswarm/azure-operator/master/service/arm_templates"
+	templateVersion            = "1.0.0.0"
+)
+
+func getDeploymentNames() []string {
+	return []string{
+		clusterSetupDeploymentName,
+	}
+}
+
+func newClusterSetupDeployment(cluster azuretpr.CustomObject) (Deployment, error) {
+	var deployment Deployment
+	{
```

Comment:
Do we get any benefit in this case, given the function length, by defining an inner block?

Reply:
Good point, removed!